### PR TITLE
Add new aria-hidden toggle 

### PR DESCRIPTION
**Pull request by blackfalcon**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #54 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

The scope of this update is to include a new `ariaHidden` attribute that will allow users to remove the default `aria-hidden="true"` value. 

Other updates outlined in commits include updates to dependencies, demo HTML text support, and Github actions workflow dependency updates. 

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by blackfalcon** [comment]
The CODEOWNERS file was never processed for this PR and I had to manually add it. Not sure why that happened, but the result was that the PR fell through the cracks and was not addressed in a timely manner.

File: src/auro-icon.js [hunk 1]
```diff
@@ -7,27 +7,30 @@
...
+ * @attr {Boolean} primary - Sets the icon to use the baseline primary icon style.
+ * @attr {Boolean} emphasis - Sets the icon to use the emphasis style.
+ * @attr {Boolean} accent - Sets the icon to use the accent style.
+ * @attr {Boolean} advisory - Sets the icon to use the advisory style.
+ * @attr {Boolean} success - Sets the icon to use the success style.
+ * @attr {Boolean} error - Sets the icon to use the error style.
+ * @attr {Boolean} disabled - Sets the icon to use the disabled style.
+ * @attr {Boolean} warning - Sets the icon to use the warning style.
+ * @attr {String} ariaHidden - Set aria-hidden value. Default is `true`. Option is `false`.
+ * @attr {String} title - Set custom description of icon in use.
```
Comment (jason-capsule42) [inline src/auro-icon.js:32]: This should be removed.

File: src/auro-icon.js [hunk 2]
```diff
@@ -107,6 +110,14 @@ class AuroIcon extends BaseIcon {
...
       customColor: {
         type: Boolean
+      },
+      ariaHidden: {
+        type: String,
+        reflect: true
+      },
+      title: {
+        type: String,
+        reflect: true
```
Comment (jason-capsule42) [inline src/auro-icon.js:120]: Also delete this
```suggestion
```

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: demo/demo.md [hunk 3]
```diff
@@ -6,94 +6,70 @@ The `auro-icon` web component comes pre-configured with all the available [Auro
...
-    @media screen and (min-width: 769px) {
-      .responsiveExample {
-        width: 30%;
-      }
-    }
-  </style>
-
-  <auro-icon class="responsiveExample" label warning category="in-flight" name="flight-delayed">Your flight's arrival has been delayed. We apologize for the delay, but your aircraft was delayed due to a late inbound aircraft.</auro-icon>
-
+  <auro-icon label category="in-flight" name="wifi">Your flight is enabled with super fast Wi-Fi</auro-icon>
```
[the remark this answers is not in the document]
Comment (blackfalcon) [reply demo/demo.md]: Addressed

File: src/auro-icon.js (hunk 2 above)
Comment (blackfalcon) [reply src/auro-icon.js]: removed

File: src/auro-icon.js (hunk 1 above)
Comment (blackfalcon) [reply src/auro-icon.js]: removed

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**